src/handbook/peopleops/hiring/reference-check.md

```diff
@@ -0,0 +1,54 @@
+---
+navTitle: Reference Check
+---
+
+Reference checks are a much needed final stage in the hiring process, conducted
+after the interview stages but before extending an offer. These conversations
+provide valuable insights into a candidate's past performance, work style, and
+potential fit with FlowFuse.
+
+## When to Conduct Reference Checks
+
+Reference checks should be conducted after all interviews are complete and the
+hiring team has identified a preferred candidate. We request 3 professional
+references from the candidate, preferably including:
+- A former direct manager
+- A peer or colleague
+- A direct report (if applicable for the role)
+
+FlowFuse will conduct 1 or 2 reference check calls, typically prioritizing
+conversations with former managers who can speak to the candidate's performance.
```

Comment:
```suggestion
We request references to ensure a range of perspectives. In cases where providing three references is not feasible (for example, candidates who are currently employed or early in their career), the hiring team may proceed with fewer references and treat this as contextual information rather than a blocker.
```